### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by loafdimension** [loafdimension/fruitful-discussions#14]
Consider the following:

Have you used semantic HTML as much as possible, not just a long list of divs?
Do all images have an alt text?
Have you used buttons for actions and links to take users to another place?
Do inputs have correctly connected labels?
Are form events handled with onSubmit and not onClick to allow for use of the keyboard?
Users should be able to navigate your site with a keyboard only - any interactive elements should be accessible, e.g. don't try to use a tag like <a> or <div> to make a button without adding the proper ARIA that's needed - see point one in this accessibility list!
If using <bold> or <italic>, replace these with <strong> and <em> respectively.
Make sure you are not doing header stepping, i.e. don't have a h2 followed by a h1, but rather a h1, then a h2, then a h3, etc.
Consider the use of ARIA labels
